modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -331,14 +303,83 @@ TEST_F(ROIList, TestInfer2)
     cv::GComputation comp(cv::GIn(in, rr), cv::GOut(age, gender));
 
     auto pp = cv::gapi::ie::Params<AgeGender> {
-        m_model_path, m_weights_path, "CPU"
+        params.model_path, params.weights_path, params.device_id
     }.cfgOutputLayers({ "age_conv3", "prob" });
     comp.apply(cv::gin(m_in_mat, m_roi_list),
                cv::gout(m_out_gapi_ages, m_out_gapi_genders),
                cv::compile_args(cv::gapi::networks(pp)));
     validate();
 }
 
+TEST(DISABLED_TestTwoIENNPipeline, InferBasicImage)
```

Comment:
This tests needs to be reworked.

The devices you use for inference may become test parameters. and not hardcoded here.
We can keep this test running by default on CPU.

MYX and GPU and other possible paths will be disabled by default.

No need to do it right now but please consider at for future.

Reply:
OK, agree with you. Honestly, all the tests should be reworked that way in the future

Thank you!